api/db/api.db.changelog.yml

```diff
@@ -30,4 +30,15 @@ databaseChangeLog:
                   constraints:
                     nullable: false
             tableName: policy_json
+   - changeSet:
+      id: 2
+      author: fujitsu
+      changes:
+      - update:
+          columns:
+          - column:
+              name: guid
+              type: varchar(50)
+              value: "1"
```

Comment:
I guess if the data is synchronized then it probably OK....
While to keep the history data synchronized too, we need to add different ids to force the synchronization.

Need confirmation from @kanekoh and @ghaih